tests/jballerina-unit-test/src/test/resources/test-src/balo/test_balo/constant/map-literal-constant-panic.bal

```diff
@@ -103,35 +95,35 @@ function getByteMap() returns map<byte> {
 // -----------------------------------------------------------
 
 function updateConstantByteMapValueInArrayWithExistingKey() {
-    foo:getBYTEA1()[0].bytem11k = 8; // panic
+    foo:getBYTEA1()[0]["bytem11k"]   = 8; // panic
 }
 
 function updateConstantByteMapValueInArrayWithNewKey() {
-    foo:getBYTEA1()[0].newKey = 8; // panic
+    foo:getBYTEA1()[0]["newKey"] = 8; // panic
```

Comment:
we shouldn't support this syntax.  A function call is not allowed in LHS. Please create an issue, This is an L1. 
```
lvexpr :=
   variable-reference-lvexpr
   | field-access-lvexpr
   | member-access-lvexpr
variable-reference-lvexpr := variable-reference
field-access-lvexpr := lvexpr . field-name
member-access-lvexpr := lvexpr [ expression ]
```

Reply:
Fix and validation added to prohibit invocation expr in lhs of an assignment. Validation is already there for compound assignment.